test/fgpu/test_katcp_interface.py

```diff
@@ -218,7 +218,8 @@ def parse_delay_string(delay_str: str) -> tuple[float, float, float, float]:
             sensor_values_str = sensor_reading[1:-1].split(",")[1:]  # Drop the timestamp
             sensor_values = (float(field.strip()) for field in sensor_values_str)
 
-            for actual_value, expected_value in zip(sensor_values, parse_delay_string(correct_delay_strings[pol])):
+            expected_values = parse_delay_string(correct_delay_strings[pol])
+            for actual_value, expected_value in zip(sensor_values, expected_values, strict=True):
```

Comment:
Any particular reason you introduced the intermediate variable?

Reply:
The line got rather long, and when black wrapped it, it became ugly.